django_lightweight_queue/runner.py

```diff
@@ -10,7 +10,15 @@
 from .worker import Worker
 from .cron_scheduler import CronScheduler, get_config
 
-def runner(log, log_filename_fn, touch_filename_fn, machine_number, machine_count, only_queue=None):
+def runner(
+        log,
+        log_filename_fn,
+        touch_filename_fn,
+        machine_number,
+        machine_count,
+        only_queue=None,
+        only_queue_workers=None,
+    ):
```

Comment:
I'm not a huge fan of reflowing long function signatures like this. In general I think it's better to break the style guide here and keep it on one line, as it's overall less messy.

(If you were to flow it like this, probably should be:
```
def fn(
    arg1,
    arg2,
):
    body
```